xla/ffi/api/api.h

```diff
@@ -1765,13 +1766,15 @@ auto DictionaryDecoder(Members... m) {
           reinterpret_cast<const XLA_FFI_Attrs*>(attr),                       \
           internal::StructMemberNames(__VA_ARGS__), diagnostic);              \
     }                                                                         \
-  }
+  };                                                                          \
+  } /* namespace xla::ffi */
```

Comment:
With your change macro is no longer an expression that is ended with `;`

In one other similar example I used static assert as the last statement in a macro to make it expression-like:

`static_assert(std::is_class_v(::xla::ffi::AttrsBinding<T>))`

Reply:
Can you please provide more details about the issue you're experiencing?

The following code snippet in `xla/ffi/ffi_test.cc` builds and runs tests successfully without any issues:
```cpp
XLA_FFI_REGISTER_STRUCT_ATTR_DECODING(::xla::ffi::PairOfI32AndF32,
                                      ::xla::ffi::StructMember<int32_t>("i32"),
                                      ::xla::ffi::StructMember<float>("f32"));
```
If there is a specific scenario where the macro is not behaving as expected, could you describe it or provide an example? This will help me better understand the problem and identify a solution.

Before my change, the macro body ended with `}`. After my change, it ends with `} /* namespace xla::ffi */`.

As per C++ best practices, macros should not end with a semicolon.  This ensures they can be used flexibly and consistently across different contexts.

Reply:
Added `static_assert` at the end of the macros